endpoints/openrtb2/amp_auction.go

```diff
@@ -191,7 +191,7 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 		LegacyLabels: labels,
 	}
 
-	response, err := deps.ex.HoldAuction(ctx, auctionRequest, nil)
+	response, err := deps.ex.HoldAuction(ctx, auctionRequest, nil, account)
```

Comment:
The account is already available in the auctionRequest. Could you please use that instead of adding a new parameter?

Reply:
Oops, I missed that. Thanks.